package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nimble-infra",
-  "version": "0.0.0",
+  "version": "0.2.5",
```

Comment:
Nope @longnd we need to test the `published` version on npm, we already published some versions to https://www.npmjs.com/package/nimble-infra :D 

We did it because we can't test the build on local at that moment, I think Hoang solved that - we can test the build on local from 0.2.9

Reply:
good to know. Thank you